lib/CMakeLists.txt

```diff
@@ -25,6 +25,13 @@ else()
 				set(ispcVersion ${curVersion})
 			endif()
 		endforeach()
+		list(APPEND ispcInstallPath ${ISPC_TEXCOMP_DIR}/ISPC/win)
+	endif()
+	if(CMAKE_SYSTEM MATCHES "Linux")
+		list(APPEND ispcInstallPath ${ISPC_TEXCOMP_DIR}/ISPC/linux)
```

Comment:
ispcInstallPath isn't set first, so would be better to use set() rather than list(APPEND).

Reply:
it doesn't matter too much actually as the cmake docs state if the var is undefined it treats it as an empty string.

Reply:
While true, it opens you up to the possibility that an earlier CMakeLists.txt defined the variable.